test/fixtures/components-models.json

```diff
@@ -125,5 +125,118 @@
         "valueType": "string"
       }
     ]
+  },
+  {
+    "id": "teaser-grouping",
+    "fields": [
+      {
+        "component": "reference",
+        "name": "image",
```

Comment:
can you group the image together with another link (just a link nothing else). That should look like

```
<p><picture><img></picture></p><p><a href="link">link</a></p>
```

if I am not mistaken. I want to see the image being properly handled in groups as well.

Reply:
added in https://github.com/mhaack/helix-importer/pull/3